gui/services/models.py

```diff
@@ -318,8 +318,7 @@ class AFP(Model):
         ),
         max_length=255,
         blank=True,
-        choices=list(choices.IPChoices()),
-        default='',
+        null=True,
```

Comment:
If it requires the creation of migration file, then I think I should  just revert this change.
Thanks for correcting me here.